### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by alissonlauffer**

### CODE REVIEW

**Review by iiiiii1wepfj — APPROVED** [review]
:+1:

**Review by HitaloSama — APPROVED** [review]

**Comment by iiiiii1wepfj** [comment]
so now this is good?

**Comment by iiiiii1wepfj** [comment]
now this is good

File: locales/pt-BR/welcome.json [hunk 1]
```diff
@@ -10,5 +10,6 @@
...
...p_msg": "A mensagem pode ser formatada no formato html ou no formato de texto embutido no Telegram. Você pode usar o comando /html para verificar se seu html está correto.\n\nVocê pode usar esses argumentos para a mensagem de boas vindas:\n - <code>{id}</code> para mostrar o ID de usuário na mensagem de boas-vindas.\n - <code>{username}</code> para exibir o nome de usuário na mensagem de boas-vindas. se o usuário não tiver um nome de usuário o bot mostrará uma menção.\n - <code>{first_name}</code> para exibir o nome do usuário na mensagem de boas vindas.\n - <code>{full_name}</code> ou <code>{name}</code> para exibir o nome completo do usuário na mensagem de boas vindas.\n - <code>{mention}</code> menciona o usuário na mensagem de boas-vindas.\n - <code>{chat_title}</code> ou <code>{title}</code> para mostrar o título do bate-papo na mensagem de boas-vindas.\n - <code>{count}</code> para exibir a contagem de membros do chat na mensagem de boas-vindas.\n\nPara adicionar botões, você pode usar <code>[botão de texto](buttonurl:link)</code>. Para adicionar mais botões na mesma linha, você pode adicionar :same após o link, como este: <code>[botão de texto](buttonurl:link:same)</code>.",
```
Comment (AndrielFR) [inline locales/pt-BR/welcome.json:13]: Bes.

**Review by AndrielFR — APPROVED** [review]

File: locales/pt-BR/welcome.json (hunk 1 above)
Comment (iiiiii1wepfj) [reply locales/pt-BR/welcome.json:13]: @AndrielFR what?

Comment (AndrielFR) [reply locales/pt-BR/welcome.json:13]: ?

Comment (iiiiii1wepfj) [reply locales/pt-BR/welcome.json:13]: @AndrielFR i dont know, i just wait for alisson for merge the pr

Comment (AndrielFR) [reply locales/pt-BR/welcome.json:13]: '-'

**Pull request merged**